lbrynet/extras/daemon/Daemon.py

```diff
@@ -2942,6 +2944,179 @@ def jsonrpc_routing_table_get(self):
         result['node_id'] = hexlify(self.dht_node.protocol.node_id).decode()
         return result
 
+    COMMENT_DOC = """
+    Comment Data Management
+    """
```

Comment:
`Data` is a redundant word here, just say "Comment management."

Reply:
Should say "Comment management." to be consistent with the others, this string is displayed as a help/comment for command groups in the cli.